File: Bai_23/main.cpp
#include <iostream>
#include <fstream>
#include <vector>

using namespace std;

int main()
{
    ifstream infile("INPUT.INP");
    ofstream outfile("OUTPUT.OUT");

    int n;
    infile >> n;

    vector<int> divisors;
    for (int i = 1; i <= n; i++)
        if (n % i == 0)
            divisors.push_back(i);

    for (int i : divisors)
        outfile << i << '\n';

    infile.close();
    outfile.close();
}

File: Bai_23/main.py
with open('E:\Language\Baitaptonghop\Bai_23\INPUT.INP','r') as file:
    n = int(file.readline().strip())
    divisors = []    
    for i in range(1, n + 1):
        if n % i == 0:
            divisors.append(i)  
    divisors.sort()

with open('E:\Language\Baitaptonghop\Bai_23\OUTPUT.OUT','w') as file_out:
    for i in divisors:
        file_out.write(str(i) + '\n')